package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solid/cli",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Could you remove the version bump? (also in package-lock).
I have a process for versioning.